### ISSUE TITLE
Cannot jump to a numbered bookmark from another file

### ISSUE THREAD

**Issue by lshlyapnikov** [alefragnani/vscode-numbered-bookmarks#37]
Steps to reproduce:
1. Set a bookmark 1 in a file A.js (Ctr+Shift+1)
2. Move cursor to the first line in the file, hit Ctr+1 (Jump to bookmark 1), it brings you to the first bookmark location
3. Switch to a file B.js
3.  Hit Ctr+1, -- nothing happens

- OS: Linux (Fedora release 27 (Twenty Seven))
- Visual Studio Code:
Version 1.22.2
Commit 3aeede733d9a3098f7b4bdc1f66b63b0f48c1ef9
Date 2018-04-12T16:34:44.222Z
Shell 1.7.12
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64
- Project type: JavaScript, no additional JavaScript extension installed

**Comment by alefragnani** [comment]
Hi @lshlyapnikov ,

By default, the bookmark navigation is only for the current file. To work the way you expect, you must change `numberedBookmarks.navigateThroughAllFiles` setting, to use `replace` or `allowDuplicates`.

Please, take a look at https://github.com/alefragnani/vscode-numbered-bookmarks#available-settings for more info.

Hope this helps

**Comment by lshlyapnikov** [comment]
Hi @alefragnani !
`"numberedBookmarks.navigateThroughAllFiles":"replace"` did exactly what I was looking for. I would say this should be the default setting, that is the _Delphi style_ behavior :)
Thanks a lot!

**Comment by alefragnani** [comment]
Hi @lshlyapnikov ,

That's great to hear it worked 🎉 .

I don't know which versions you have used, or which plugins you have installed, but I never saw this behavior in Delphi. To me, bookmarks has always been _per file_. That's the reason I created the extension this way.

Nevertheless, I added this setting one year go, after #6 has been reported. I added two new behaviors based on the request and bit of research on others IDEs. I guess it goes _beyond Delphi_ today, but I still have **Delphi** in mind 😁 .

Thanks
